src/RestControllers/AuthRestController.php

```diff
@@ -110,7 +110,7 @@ public function authenticate($authPayload)
             return;
         }
         $encoded_token = base64_encode(json_encode(['token' => $encrypted_new_full_token, 'site_id' => trim($_SESSION['site_id']), 'api' => trim($_SESSION['api'])]));
-        $give = array("token_type" => "Bearer", "access_token" => $encoded_token, "expires_in" => "3600");
+        $give = array("token_type" => "Bearer", "access_token" => $encoded_token, "expires_in" => "3600", "user_id" => $userId);
```

Comment:
Makes sense. However it would be prudent to make this a data array for future if somebody wants additional info returned.
 "user_data" =>array("user_id" => $userId)

Reply:
like this idea. to clarify, this is all json formatting correct?

Reply:
answered my own question above :)
HttpRestRouteHandler json's it